src/app/dashboard/components/exoft-achievements/exoft-achievements.component.html

```diff
@@ -1,28 +1,33 @@
 <div class="exoftachiv">
+
   <div class="tittle">
     <span class="name">EXOFT ACHIEVEMENTS</span>
     <div class="achievement-button">{{exoftAchievements.length}}</div>
   </div>
+
   <div class="content">
     <mat-list class="list">
       <mat-list-item *ngFor="let exoftAchievement of exoftAchievements">
         <div class="list-content">
+
           <div class="first-block">
```

Comment:
`<div class="list-content">` and `<div class="first-block"> `are not one-level tags. you can remove epty line here

Reply:
fixed